openassessment/templates/openassessmentblock/response/oa_response_cancelled.html

```diff
@@ -32,12 +32,12 @@ <h5 class="message__title">{% trans "Status" %}</h5>
                     {% trans "Your submission was cancelled. " %}
                     <p>
                         {% if workflow_cancellation.cancelled_by %}
-                            {% blocktrans with removed_datetime=workflow_cancellation.cancelled_at|timezone:time_zone|date:"N j, Y H:i e" removed_by_username=workflow_cancellation.cancelled_by %}
-                                Your submission has been cancelled by {{ removed_by_username }} on {{ removed_datetime }}
+                            {% blocktrans with removed_datetime=workflow_cancellation.cancelled_at|timezone:"UTC"|date:"N j, Y H:i e" removed_by_username=workflow_cancellation.cancelled_by %}
+ -                                Your submission has been cancelled by {{ removed_by_username }} on {{ removed_datetime }}
                             {% endblocktrans %}
                         {% else %}
-                            {% blocktrans with removed_datetime=workflow_cancellation.cancelled_at|timezone:time_zone|date:"N j, Y H:i e" %}
-                                Your submission was cancelled on {{ removed_datetime }}
+                            {% blocktrans with removed_datetime=workflow_cancellation.cancelled_at|timezone:"UTC"|date:"N j, Y H:i e" %}
+ -                              Your submission was cancelled on {{ removed_datetime }}
```

Comment:
It isn't -- previously it was a simple datestamp style string (e.g. '2014-12-25 14:46 UTC') - and I was unsure about making a UX change. I will follow up with product about this.